### PR TITLE
Remove an invalid import statement

### PR DESCRIPTION
**Pull request by mjeronimo**
Also, don't continue if File Open dialog box was canceled.

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

### CODE REVIEW

File: rqt_bag/src/rqt_bag/bag_widget.py [hunk 1]
```diff
@@ -285,6 +284,8 @@ def _handle_load_clicked(self):
                 self.last_open_dir = filenames[0]
             for filename in filenames:
                 self.load_bag(filename)
+        else:
+            return
```
Comment (ivanpauno) [inline rqt_bag/src/rqt_bag/bag_widget.py:288]: I would rather invert the logic like:
```python3
        if not dialog.exec():
            return
        filenames = dialog.selectedFiles()
        if filenames:
            self.last_open_dir = filenames[0]
        for filename in filenames:
            self.load_bag(filename)
```

**Review by ivanpauno — APPROVED** [review]
LGTM!

File: rqt_bag/src/rqt_bag/bag_widget.py [hunk 2]
```diff
@@ -42,7 +42,6 @@
 from python_qt_binding.QtGui import QIcon, QResizeEvent
 from python_qt_binding.QtWidgets import QFileDialog, QGraphicsView, QWidget
 
-from rosbag2_transport import rosbag2_transport_py
```
Comment (clalancette) [inline rqt_bag/src/rqt_bag/bag_widget.py:45]: If we're removing this, should we also remove the dependency? https://github.com/ros-visualization/rqt_bag/blob/2049032e6fbad93463f9b626c997292f08cac62e/rqt_bag/package.xml#L19

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rqt_bag/src/rqt_bag/bag_widget.py (hunk 2 above)
Comment (mjeronimo) [reply rqt_bag/src/rqt_bag/bag_widget.py:45]: Yes, good catch. I'll update.

**Review by clalancette — APPROVED** [review]
Looks good to me with green CI.

**Pull request merged**